### PR TITLE
50389_book_index_user

### PR DESCRIPTION
**Pull request by noor-framgia**

### CODE REVIEW

File: app/controllers/books_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class BooksController < ApplicationController
+
+  def index
+  end
+
+  def show
+  end
+
```
Comment (abdullahallmehedi) [inline app/controllers/books_controller.rb]: remove

File: app/controllers/books_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class BooksController < ApplicationController
+
```
Comment (abdullahallmehedi) [inline app/controllers/books_controller.rb]: remove

**Comment by SSalekin** [comment]
ok

**Comment by Rownaknira** [comment]
ok

**Pull request merged**